### DISCUSSION TITLE
Should new issues be created on the xarray-contrib/datatree repository?

### DISCUSSION THREAD

**Discussion by eschalkargans** [pydata/xarray#9030] [General] [answered]
Hello,

I recently created multiple new issues on the datatree repository:https://github.com/xarray-contrib/datatree/issues

Indeed, I am using an old version of xarray (xr.__version__ == 2023.10.1), and keep using the datatree package separately (dt.__version__ == '0.0.13'). 

However, the activity on this repository seems to have diminished ; should we now use datatree from xarray directly?

I did not follow the datatree merging into xarray lately so I am not sure of what the status is. 

Thank you!

**Comment by TomNicholas** [comment, marked as answer]
Hi @eschalkargans ! I see you raising issues and suggestions on the datatree repository which is amazing 🙏 

> I did not follow the datatree merging into xarray lately so I am not sure of what the status is.

We have merged almost the entire datatree code into xarray, but not made it public yet as we haven't merged the docs. See the issue tracking progress here https://github.com/pydata/xarray/issues/8572

> should we now use datatree from xarray directly?

We are almost but not quite at the point of making this public. If you wanted to be the intrepid first beta-tester of the version of datatree that's in xarray then that would be super helpful.

> I recently created multiple new issues on the datatree repository

These issues are super helpful, and just raising them anywhere is great so that they are recorded. Our priority right now is to get a minimum version of datatree released in xarray, and then we will have more time to come back and implement whatever fixes and improvements have been suggested in the meantime. That said, if you want to skip a step and just submit issues / bugfixes / PRs directly the datatree code in xarray that works too!

**Reply by dcherian** [reply]
Presumably you can import these issues into `pydata/xarray` and archive this repo, correct?

**Reply by TomNicholas** [reply]
Oh yeah if we can just import them over that would save us having to remember to go back.

**Reply by eschalkargans** [reply]
Thanks for your answer!

> We are almost but not quite at the point of making this public. If you wanted to be the intrepid first beta-tester of the version of datatree that's in xarray then that would be super helpful.

I work in a project with a fixed version of xarray, so for now I still have to use an older version of xarray, and datatree as separate package. But once the upstream dependencies are updated, I can start using the datatree from xarray directly!

Regarding issues creation I will simply use this strategy:

- If I use datatree from the standalone package, I will create the issues in the datatree repo
- In the future when I can use datatree from xarray, I will create the issues in the xarray repo

> Our priority right now is to get a minimum version of datatree released in xarray, and then we will have more time to come back and implement whatever fixes and improvements have been suggested in the meantime. That said, if you want to skip a step and just submit issues / bugfixes / PRs directly the datatree code in xarray that works too!

So I understand that, once the MVP version of datatree is released into xarray, it is time to make all new developments / issues / MRs into xarray directly and stop using datatree.

Regarding the PRs on datatree, I had personally 2 of them open currently ; however they were more about small bugfixes, not urgent and not related a MVP ; is it planned to perform a pass over the remaining 21 PRs of the datatree repo once the MVP datatree is released with xarray?
